actors/verifreg/src/types.rs

```diff
@@ -34,6 +38,10 @@ pub struct BytesParams {
 pub type UseBytesParams = BytesParams;
 pub type RestoreBytesParams = BytesParams;
 
+pub const SIGNATURE_DOMAIN_SEPARATION_REMOVE_DATA_CAP: &str = "fil_removedatacap:";
```

Comment:
```suggestion
pub const SIGNATURE_DOMAIN_SEPARATION_REMOVE_DATA_CAP: &[u8] = b"fil_removedatacap:";
```

(given that we always use this as "bytes"